### ISSUE TITLE
Accessibility to R Conferences and Events for Community Members with Visual Impairments

### ISSUE THREAD

**Issue by DragonflyStats** [forwards/Community#10]
The issue of accessibility for conferences for prospective attendees for visual impairments was raised on social media, in the wake of UseR! 2019.

The fact that conferences functionally exclude the visually impaired through lack of adequate accessibility, and consequently their perspectives were not accounted for in post-conference surveys was highlighted.

A task for the coming year, in anticipation of conferences in 2020, would be to work with conference organizers to devise action plans to make the conferences more accessible.

**Comment by hturner** [comment]
I think we've got this covered in these issues: 

 - update survey: https://github.com/forwards/tasks/issues/51
 - update event best practices: https://github.com/forwards/tasks/issues/50
